pkg/kotsadmsnapshot/backup.go

```diff
@@ -275,20 +269,20 @@ func CreateInstanceBackup(ctx context.Context, cluster *downstreamtypes.Downstre
 	}
 
 	logger.Infof("Creating instance backup CR %s", veleroBackup.GenerateName)
-	backup, err := veleroClient.Backups(metadata.backupStorageLocationNamespace).Create(ctx, veleroBackup, metav1.CreateOptions{})
+	err = ctrlClient.Create(ctx, veleroBackup)
```

Comment:
Namespace needs to be set in `getInfrastructureInstanceBackupSpec`:
- https://github.com/replicatedhq/kots/blob/a422a554bd22d7b1e7e0feed4f76e33d77f0fc57/pkg/kotsadmsnapshot/backup.go#L435